pkg/pipelines/bootstrap.go

```diff
@@ -40,6 +41,11 @@ type BootstrapOptions struct {
 
 // Bootstrap bootstraps a GitOps pipelines and repository structure.
 func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
+	exists, err := ioutils.IsExisting(appFs, filepath.Join(o.OutputPath, pipelinesFile))
```

Comment:
"pipelinesFile" can be set  by --pipelines-file flag.   You can't just use the constant.

Reply:
Will be making the name of the pipelines.yaml as constant in consideration of the UI

Reply:
Yeah, we pretty much need the file to be called `pipelines.yaml` so that we can fetch it from the upstream GitOps repo when we're rendering the UI.